extensions/sql-migration/src/dialog/assessmentResults/sqlDatabasesTree.ts

```diff
@@ -22,7 +23,7 @@ const styleRight: azdata.CssStyles = {
 };
 
 export class SqlDatabaseTree {
-
+	private _view!: azdata.ModelView;
```

Comment:
This doesn't really make sense as a class member - why not just pass it into the functions that need it like you already are in most of the places?

Reply:
I was planning on removing that function argument. It there a benefit of doing this?
I guess testing, right?

Reply:
It's not a huge deal either way - but this isn't generally something that I would expect to persist beyond the initial creation of the components so I would usually just pass it in as an argument in that case. 

(we have examples of both ways so do whatever one makes sense to you, but I agree it should at least be consistent)